GLib-2.0.gir

```diff
@@ -12890,32 +12890,6 @@ module.</doc>
       <doc xml:space="preserve">Multiplying the base 2 exponent by this number yields the base 10 exponent.</doc>
       <type name="gdouble" c:type="gdouble"/>
     </constant>
-    <constant name="LOG_DOMAIN" value="0" c:type="G_LOG_DOMAIN">
```

Comment:
Instead this can be ignored in the Gir.toml file which will drop it from the ffi tests

Reply:
I tried that but couldn't get it to work, let me try again

Reply:
we do that in gtk for the version constants for exampel